test/tools/isoltest.cpp

```diff
@@ -422,8 +422,9 @@ int main(int argc, char const *argv[])
 		{
 			auto options = std::make_unique<solidity::test::IsolTestOptions>();
 
-			if (!options->parse(argc, argv))
-				return -1;
+			bool shouldContinue = options->parse(argc, argv);
+			if (!shouldContinue)
+				return 0;
```

Comment:
I'm not sure I understand. It used to return -1 if you used `--help`. I think this is a case that should result in success and not an error.

Good suggestion about `EXIT_SUCCESS` instead of `0`. I'll change that.